test/hook/context/context_combiner.go

```diff
@@ -0,0 +1,90 @@
+package context
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/flant/shell-operator/pkg/hook/binding_context"
+	"github.com/flant/shell-operator/pkg/hook/controller"
+	"github.com/flant/shell-operator/pkg/hook/task_metadata"
+	"github.com/flant/shell-operator/pkg/hook/types"
+	shell_operator "github.com/flant/shell-operator/pkg/shell-operator"
+	"github.com/flant/shell-operator/pkg/task"
+	"github.com/flant/shell-operator/pkg/task/queue"
+)
+
+const TestTaskType task.TaskType = "TestTask"
+
+type ContextCombiner struct {
+	op *shell_operator.ShellOperator
+	q  *queue.TaskQueue
+}
+
+func NewContextCombiner() *ContextCombiner {
+	op := &shell_operator.ShellOperator{}
+	op.TaskQueues = queue.NewTaskQueueSet()
+	op.TaskQueues.WithContext(context.Background())
+	op.TaskQueues.NewNamedQueue("main", nil)
```

Comment:
I did not get why there is the "main" queue now. Previously we did not specify any queue. And it is better to make `"main"` constant like `"TestTask"`.